### PR TITLE
Prettier rules

### PR DESCRIPTION
**Pull request by jeannsensei**
I added the prettier settings for the linting of the code, I also added some information in the readme page of the project.
Maybe we can come to terms later with whats the best "printWidth" number for example, but I used those settings since is similar from what I saw in the DAT project.

### CODE REVIEW

**Review by dieguisherrera — CHANGES_REQUESTED** [review]
It's quick confusing to have the folder `angular-marketplace` inside client, I think it would be better if we just have directly the angular code there

File: client/angular-marketplace/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (dieguisherrera) [inline client/angular-marketplace/.prettierrc:1]: @jeannsensei I have a doubt here, whta happen if this rules doesn't match the Visual Studio formating? I think it woul be better if we create a fault tolerance set of rules at the beggining, or share the VS configuration file that match this cases (e.g. https://github.com/nitin27may/mean-docker/blob/master/.vscode/settings.json)

File: client/angular-marketplace/README.md [hunk 2]
```diff
@@ -23,6 +23,34 @@ cd client/angular-marketplace
...
+  "singleQuote": true,
+  "printWidth": 300,
+  "html.format.wrapLineLength": 300,
+  "html.format.wrapAttributes": "auto"
+}
+```
+
+The file .prettierrc is placed outside the /src folder, we can format all the .ts files by running:
+```BASH
+npm run prettier-format
```
Comment (dieguisherrera) [inline client/angular-marketplace/README.md:50]: I will propose to automatize all of this in a docker file later. The idea would be that in the root we have a docker-compose that use this folder to run the angular container (not a image), and have this automatized. Once the Angular installation was finished, I can help with that

File: client/angular-marketplace/README.md [hunk 3]
```diff
@@ -23,6 +23,34 @@ cd client/angular-marketplace
...
+```
+
+
 ## Development server
```
Comment (dieguisherrera) [inline client/angular-marketplace/README.md:54]: I'm creating this parts a services for docker-compose

**Review by dieguisherrera — CHANGES_REQUESTED** [review]

**Review by nrodriguez11 — APPROVED** [review]

**Comment by jeannsensei** [comment]
> It's quick confusing to have the folder `angular-marketplace` inside client, I think it would be better if we just have directly the angular code there

I was about to do that but in another PR... should I do it here since we're talking about it?

File: client/angular-marketplace/.prettierrc (hunk 1 above)
Comment (jeannsensei) [reply client/angular-marketplace/.prettierrc:1]: Good suggestion. Let me check that and I'll update.

File: client/angular-marketplace/README.md (hunk 2 above)
Comment (jeannsensei) [reply client/angular-marketplace/README.md:50]: Yeah, we can do that later! I'll write it down.

File: client/angular-marketplace/.prettierrc (hunk 1 above)
Comment (jeannsensei) [reply client/angular-marketplace/.prettierrc]: Let's tackle this in another PR.

**Comment by jeannsensei** [comment]
We will tackle in another PR the vscode set of rules at the beggining, or share the VS configuration file that match this cases (e.g. https://github.com/nitin27may/mean-docker/blob/master/.vscode/settings.json)

**Comment by davidtorroija** [comment]
I like this PR it has good communication

**Pull request merged**